app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -130,15 +131,57 @@ export function addSaveableUserOptionField(fieldName) {
   userOptionFields.push(fieldName);
 }
 
+function userOption(userOptionKey) {
+  return computed(`user_option.${userOptionKey}`, {
+    get(key) {
+      deprecated(
+        `Getting ${key} property of user object is deprecated. Use user_option object instead`,
+        { id: "discourse.user.userOptions" }
```

Comment:
I think it'll also be useful to set `since` and `dropFrom`. In the testing environment, I think it'll be helpful if we set `raiseError` to `true`.

Reply:
I do not think we ever set `raiseError: true` for any of the other deprecation notices. Maybe we should set that when we get close to `dropFrom` time?